lib/github_api/client/git_data/tags.rb

```diff
@@ -4,6 +4,7 @@ module Github
   class Client::GitData::Tags < API
     # This tags api only deals with tag objects -
     # so only annotated tags, not lightweight tags.
+    # Refer https://developer.github.com/v3/git/tags/#parameters
 
     VALID_TAG_PARAM_NAMES = %w[
```

Comment:
@piotrmurach I think rather than removing it altogether, we should validate the json schema using something like [json-schema](https://github.com/ruby-json-schema/json-schema) since, Github is going to allow a specific number of requests and IMO we shouldn't allow the user to send badly formated data.